### PR TITLE
Finished Problems 37 and 38.

### PR DESCRIPTION
**Pull request by ikhat**
Wasn't 100% sure what you are looking for with the last part of 37, about which coordinates better represent the point, so I elaborated a bit.

Also wasn't sure if you wanted a detailed explanation of how to find change of basis matrices. 

Let me know.

### CODE REVIEW

File: problemsets-mat223/linearalbegra.tex [hunk 1]
```diff
@@ -2566,25 +2566,42 @@ \section*{Change of Basis \& Coordinates}
...
+		\item The points $X=(4,3)$ and $Y=(1,7)$ are given in east-north coordinates. 
+			Find their street coordinates.
+			\begin{solution}[inline]
+				$X=(5,0)$ and $Y=(5,5)$ in street coordinates.
+			\end{solution}
 		\item Define $\vec e_1=\mat{1\text{ east}\\0\text{ north}}$ and 
-			$\vec e_2=\mat{0\text{ east}\\1\text{ north}}$. Does $\Span\{\vec e_1,\vec e_2\}
-			=\Span\{\vec d_1,\vec d_2\}$?
+			$\vec e_2=\mat{0\text{ east}\\1\text{ north}}$.
+			Does $\Span\{\vec e_1,\vec e_2\} = \Span\{\vec d_1,\vec d_2\}$?
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: `\Set`

File: problemsets-mat223/linearalbegra.tex [hunk 2]
```diff
@@ -2601,24 +2618,52 @@ \section*{Change of Basis \& Coordinates}
...
+				\[
+					\vec v=2\vec e_1+2\vec e_2
+					=2(3\vec c_1-\vec c_2)+2(-5\vec c_1+2\vec c_2)
+					=-4c_1+2c_2.
+				\]
+			\end{solution}
+		\item Can you find a matrix $X$ so that $X[\vec w]_{\mathcal C} = [\vec w]_{\mathcal E}$
+			for any	$\vec w$?
+			\begin{solution}[inline]
+				$X = \mat{2&5\\1&3}$ is such a matrix.
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: Explain how you got `X` and `Y`.

File: problemsets-mat223/linearalbegra.tex [hunk 3]
```diff
@@ -2647,18 +2647,50 @@ \section*{Change of Basis \& Coordinates}
...
 				\]
 			\end{solution}
 		\item Can you find a matrix $X$ so that $X[\vec w]_{\mathcal C} = [\vec w]_{\mathcal E}$
 			for any	$\vec w$?
-			\begin{solution}[inline]
-				$X = \mat{2&5\\1&3}$ is such a matrix. 
+			\begin{solution}
+				$X = \mat{2&5\\1&3}$ is such a matrix.
+
+				For a vector $\vec w$, if $[\vec w]_{\mathcal C}=\mat{x\\y}$ then
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: I would like this proof to go as follows: 

$X$ is 2x2, so $X=[a b; c d]$ for some a,b,c,d\in R. We know [..]_C=[..]_E and so X[..]_C=[..]_E. Similarly, [..']_C=[..']_E
and so X[..']_C=[..']_E. We can now solve for a,b,c,d to get...

Comment (ikhat) [reply problemsets-mat223/linearalbegra.tex]: Ah. Do you mean you want those relationships to be between the basis vectors? I mean something like:

X=[a b; c d] for some a,b,c,d\in R, and we know 

[1 0]_C = [2 1]_E and [0 1]_C = [5 3]_E. 

So we need 

X[1 0]_C = [a c] = [2 1]_E and X[0 1]_C = [b d] = [5 3]_E

Comment (siefkenj) [reply problemsets-mat223/linearalbegra.tex]: That's what I mean!

File: problemsets-mat223/linearalbegra.tex [hunk 4]
```diff
@@ -2601,24 +2618,96 @@ \section*{Change of Basis \& Coordinates}
...
+
+				We know from the first part above that
+				\[
+					\mat{1\\0}_{\mathcal C} = \mat{2\\1}_{\mathcal E}
+					\quad \text{and} \quad
+					\mat{0\\1}_{\mathcal C} = \mat{5\\3}_{\mathcal E},
+				\]
+				and that we need $X$ to satisfy
+				\[
+					X\mat{1\\0}_{\mathcal C}=\mat{a\\c}=\mat{2\\1}_{\mathcal E}
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: The notation isn't right here. Drop the subscripts.  [...]_C is a true vector and [...]_C=[...]_E. You've forced X to be the identity transformation.

File: problemsets-mat223/linearalbegra.tex [hunk 5]
```diff
@@ -2601,24 +2618,96 @@ \section*{Change of Basis \& Coordinates}
...
+			for any	$\vec w$?
+			\begin{solution}
+				$X = \mat{2&5\\1&3}$ is such a matrix.
+
+				We know $X$ must be a $2\times2$ matrix, so suppose $X=\mat{a&b\\c&b}$
+				for some $a,b,c,d\in\R$. 
+
+				We know from the first part above that
+				\[
+					\mat{1\\0}_{\mathcal C} = \mat{2\\1}_{\mathcal E}
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: This should be rephrased like: "From the previous part, we know X[1 0] = [2 1] and ..."

File: problemsets-mat223/linearalbegra.tex [hunk 6]
```diff
@@ -2601,24 +2618,90 @@ \section*{Change of Basis \& Coordinates}
...
+
+				From the first part above, we know
+				\[
+					\mat{1\\0}_{\mathcal C} = \mat{2\\1}_{\mathcal E}
+					\quad \text{and} \quad
+					\mat{0\\1}_{\mathcal C} = \mat{5\\3}_{\mathcal E},
+				\]
+				and so we need $X$ to satisfy
+				\[
+					X\mat{1\\0}=\mat{a\\c}=\mat{2\\1}
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: You don't *need* X to satisfy X[1 0] = [a c]. This is just a fact. Please reword.

Comment (ikhat) [reply problemsets-mat223/linearalbegra.tex]: Sure, I can do that.

**Pull request merged**